### DISCUSSION TITLE
サービスの作成ができない

### DISCUSSION THREAD

**Discussion by karszawa** [topotal/waroom-roadmap#16] [Bug Report]
こんにちは。ベータ版をトライアルとして利用させていただいている dinii 社の karszawa と申します。Waroom の予期せぬ挙動によりトライアルを実施できないため事象をご報告させていただきます。ご確認いただけますと幸いです。

## 事象

下記のページにて新規のサービスを作成しようとしたのですが、api.beta.waroom.com/graphql より 500 エラーが返されるようでサービスの作成を完了できません。いくつかの文字列を試してみましたが同様の結果でした。

https://beta.waroom.com/organizations/dinii/services

## 詳細

時刻: 2022-09-08 10:30:00 頃
x-request-id: 1fb12dd5-e8b6-43d9-9b01-af94d60809bb

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/1257695/189014624-68b2f3ff-05e7-4dbe-925e-c4d3aabb7c70.png">

## 補足

ログイン後にこちらと似たような事象が発生したため、もしかすると認証状態に関するエラーかもしれません。 → https://github.com/topotal/waroom-roadmap/discussions/8

ちなみに、ログイン後に遷移する https://beta.waroom.com/organizations という URL では依然として 500 エラーが発生しており、直に https://beta.waroom.com/organizations/dinii/incidents を開いて検証をしているという状況でした。

**Comment by sawa-zen** [comment]
@karszawa さま
フィードバックありがとうございます！確認いたします 🙏🏻

**Reply by karszawa** [reply]
@sawa-zen 確認ありがとうございます！

ただこちら自己解決してしましました 🙏 

状況としては、アカウントを作成後に https://beta.waroom.com/organizations/dinii/incidents を開いてデータが確認できたつもりでいたのですが、実際には dinii organization に自分のアカウントが招待されていなかったためデータが見れないということのようでした。

organization のオーナーから自分のアカウントを dinii に招待してもらい、データの表示・サービスの作成が実施できることを確認いたしました。

てっきり、組織のドメインのアカウントでログインすると勝手に organization への紐付けが行われるのかと思っていたのですが勘違いだったようです 😅 

失礼いたしました 🙏

**Reply by sawa-zen** [reply]
@karszawa 
いえいえ！
こちらの問題現在対応中でして、現状ですと招待したメンバーの方には招待メールを確認していただくようアナウンスいただけますと幸いです 🙇🏻 
お手数かけてしまい申し訳ありません 🙏🏻
